### DISCUSSION TITLE
Moinalloc warning

### DISCUSSION THREAD

**Discussion by Unaimend** [ablab/spades#1099] [Q&A] [answered]
### Description of bug

Hello spades team, I am currently writing my Bsc. thesis which is due in 20 days.
I alreay  did multiple assemblies with spades but for my latest dataset, I get an error after 5 days.
```
mimalloc: warning: unable to allocate from region: size 4194304
```
but the assembly did not crash. I just wanted to let you know that this happened and I hope that this is not a big issue but I am a little worried because I did not get any log entries since then. Should I let it run?

Thank you for your time,
Thomas 

### spades.log

[output.log](https://github.com/ablab/spades/files/10896149/output.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/10896159/params.txt)

### SPAdes version

  SPAdes version: 3.15.5

### Operating System

Linux-4.18.0-372.9.1.el8.x86_64-x86_64-with-glibc2.28

### Python Version

Python version: 3.11.0

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

### 
I used slurm but the job has 600GB available

So threads a re till doing sth. but most of them stay at 0 percent
![image](https://user-images.githubusercontent.com/7808379/223068603-f5f814d6-a7c2-4a6f-be03-04566d751339.png)

**Comment by asl** [comment, marked as answer]
You can ignore these warnings, they are benign

**Reply by Unaimend** [reply]
Ok thank you very much. You just saved my day and my Thesis :)

**Reply by Unaimend** [reply]
But is it normal that I don't get any logs for more than 24 hours?

**Reply by asl** [reply]
It seems you're having plenty of reads. So some processing time is expected.
